Document/PostManager.php

```diff
@@ -96,32 +96,52 @@ public function findOneByPermalink($permalink, BlogInterface $blog)
      */
     public function getPager(array $criteria, $page, $limit = 10, array $sort = array())
     {
+        if (!isset($criteria['mode'])) {
+            $criteria['mode'] = 'public';
+        }
+
         $parameters = array();
         $query = $this->getRepository()
             ->createQueryBuilder('p')
             ->select('p, t')
             ->leftJoin('p.tags', 't')
-            ->orderby('p.publicationDateStart', 'DESC');
+            ->orderBy('p.publicationDateStart', 'DESC');
 
-        // enabled
-        $criteria['enabled'] = isset($criteria['enabled']) ? $criteria['enabled'] : true;
-        $query->andWhere('p.enabled = :enabled');
-        $parameters['enabled'] = $criteria['enabled'];
+        if (!isset($criteria['enabled']) && $criteria['mode'] == 'public') {
+            $criteria['enabled'] = true;
+        }
+        if (isset($criteria['enabled'])) {
+            $query->andWhere('p.enabled = :enabled');
+            $parameters['enabled'] = $criteria['enabled'];
+        }
 
-        if (isset($criteria['date'])) {
+        if (isset($criteria['date']) && isset($criteria['date']['query']) && isset($criteria['date']['params'])) {
```

Comment:
Annnnnd new issue is here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1977